frontend/src/features/properties/selector/search/PropertySearchSelectorFormView.test.tsx

```diff
@@ -187,54 +189,35 @@ describe('PropertySearchSelectorFormView component', () => {
 
   describe('selecting results', () => {
     it('search results can be selected', async () => {
-      const { findByTestId } = setup({ searchResults: mockPropertyLayerSearchResponse.features });
-      const checkbox = await findByTestId(
-        'selectrow-WHSE_CADASTRE.PMBC_PARCEL_FABRIC_POLY_SVW.fid-674bf6f8_180d8c9b18e_7c12',
-      );
+      const { findByTestId, container } = setup({
+        searchResults: toMapProperty(mockPropertyLayerSearchResponse.features),
+      });
+      console.log(container.outerHTML);
```

Comment:
can we remove this to avoid cluttering up the logs?

Reply:
for sure!